go/vt/vtexplain/vtexplain_vtgate.go

```diff
@@ -130,6 +131,32 @@ func (vte *VTExplain) buildTopology(ctx context.Context, opts *Options, vschemaS
 
 		vte.explainTopo.KeyspaceShards[ks] = make(map[string]*topodatapb.ShardReference)
 
+		srvPath := path.Join(topo.KeyspacesPath, ks, topo.SrvKeyspaceFile)
+		srvKeyspace := &topodatapb.SrvKeyspace{
+			Partitions: []*topodatapb.SrvKeyspace_KeyspacePartition{
+				{
+					ServedType:      topodatapb.TabletType_PRIMARY,
+					ShardReferences: shards,
+				},
+				{
+					ServedType:      topodatapb.TabletType_REPLICA,
+					ShardReferences: shards,
+				},
+				{
+					ServedType:      topodatapb.TabletType_RDONLY,
+					ShardReferences: shards,
+				},
+			},
+		}
+		data, err := srvKeyspace.MarshalVT()
+		if err != nil {
+			return err
+		}
+		_, err = conn.Update(ctx, srvPath, data, nil)
+		if err != nil {
+			return err
+		}
```

Comment:
@shlomi-noach In order to not have the throttler log errors inside `vtexplain`, I had to set up the above minimum stuff in the memory topo for `SrvKeyspace`. 

I wonder if there's a better way for this and if this could be refactored in a better way? Should we have some option to disable more parts of the throttler when we're inside `vtexplain`? 

I guess alternatively, we could make the throttler a parameter in `NewTabletServer` instead to inject it (which is called inside `vtexplain`), so we could inject a no-op throttler? Any other ideas?